node/config/types.go

```diff
@@ -31,6 +31,18 @@ type FullNode struct {
 	Index      IndexConfig
 }
 
+// Follower is a full node config
+type Follower struct {
+	Common
+	Client     Client
+	Wallet     Wallet
+	Fees       FeeConfig
+	Chainstore Chainstore
+	Cluster    UserRaftConfig
+	Fevm       FevmConfig
+	Index      IndexConfig
+}
+
```

Comment:
This shouldn't be needed, we're still starting a chain node (which in the code is called a "full node", but that's really any chain node)

Reply:
done